docs/data_fields.md

```diff
@@ -0,0 +1,28 @@
+
+The main fields are the following, plus the geographical entity fields explained below:
+
+* date: the date the data point refers to
+* type: the type of data point: cases, tested, deaths, hospitalized, discharged, or recovered
```

Comment:
maybe more detail on what these words mean exactly

Reply:
especially that they're all cumulative, and that tested means regardless of positive/negative result.

Reply:
added the word "cumulative" (good point). I would like to know a bit more for each type though, but let's not the perfect get in the way of the good.